### PR TITLE
PIL-679 - Refactor TaskList

### PR DESCRIPTION
**Pull request by TevynAllen**

### CODE REVIEW

File: app/views/TaskListView.scala.html [hunk 1]
```diff
@@ -15,120 +15,64 @@
...
+    @para(messages("taskList.paragraph"))
+    @h2(messages("taskList.subheading.complete"))
+    @para(messages("taskList.completedSections", counter), classes = "govuk-body govuk-!-margin-bottom-7")
+
 
     <ol class="app-task-list">
-            <!-- Business Section -->
-        <li>
+        <li> <!-- Group details -->
             <h2 class="app-task-list__section">
```
Comment (AKKUM) [inline app/views/TaskListView.scala.html:48]: This can also be made main section then put sub section inside it. But happy with code. So approving it. 👍

**Review by AKKUM — DISMISSED** [review]

File: app/views/components/gds/HmrcStatusTag.scala.html
Comment (aashik-varahan) [inline app/views/components/gds/HmrcStatusTag.scala.html:1]: Can we add a new line at EOF please for UNIX standard

File: app/views/components/gds/TaskListSection.scala.html
Comment (aashik-varahan) [inline app/views/components/gds/TaskListSection.scala.html:1]: Can we add a new line at EOF please for UNIX standard

Comment (TevynAllen) [reply app/views/components/gds/TaskListSection.scala.html:1]: There is

**Review by aashik-varahan — APPROVED** [review]

**Pull request merged**